File: NetFlexAPI/NetFlexAPI/Models/Episode.cs
namespace NetFlexAPI.Models;

public class Episode
{
    public Guid Id { get; set; }
    public string Title { get; set; }
    public Guid SerialId { get; set; }
    public int Duration { get; set; }
    public int Number { get; set; }
    public string VideoLink { get; set; }
    public string PreviewVideo { get; set; }
}

File: NetFlexAPI/GiraffeAPI/Models/Episode.fs
module GiraffeAPI.Models.Episode

open System
open System.ComponentModel.DataAnnotations
open Microsoft.EntityFrameworkCore.Metadata.Internal

[<CLIMutable>]  
type Episode =
    {
        Id : Guid
        Title : string
        SerialId : Guid
        Duration : int
        Number : int
        VideoLink : string
        PreviewVideo : string
    }
    
    